cmd/agent/common/path/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/DataDog/datadog-agent/cmd/agent/common/path
+
+go 1.20
+
+replace github.com/DataDog/datadog-agent/pkg/util/executable => ../../../../pkg/util/executable
+
+replace github.com/DataDog/datadog-agent/pkg/util/log => ../../../../pkg/util/log
+
+require (
+	github.com/DataDog/datadog-agent/pkg/util/executable v0.0.0-00010101000000-000000000000
+	github.com/DataDog/datadog-agent/pkg/util/log v0.49.0-rc.2
+	github.com/DataDog/datadog-agent/pkg/util/winutil v0.36.1
+	golang.org/x/sys v0.12.0
+)
+
+require (
+	github.com/DataDog/datadog-agent/pkg/util/scrubber v0.49.0-rc.2 // indirect
```

Comment:
Don't you need to `replace` `winutil` and `scrubber` too ?

Reply:
Added for scrubber

Looks like winutil hasn't been modularized yet, do we want to wait for winutil to go in first? @dineshg13

Reply:
Hum probably yes, I guess otherwise this module would depend on the whole agent...
No idea where the `v0.36.1` comes from :smile: